packages/block-editor/src/components/block-controls/groups.js

```diff
@@ -1,13 +1,20 @@
 /**
  * WordPress dependencies
  */
-import { createSlotFill } from '@wordpress/components';
+import { privateApis as componentsPrivateApis } from '@wordpress/components';
 
-const BlockControlsDefault = createSlotFill( 'BlockControls' );
-const BlockControlsBlock = createSlotFill( 'BlockControlsBlock' );
-const BlockControlsInline = createSlotFill( 'BlockFormatControls' );
-const BlockControlsOther = createSlotFill( 'BlockControlsOther' );
-const BlockControlsParent = createSlotFill( 'BlockControlsParent' );
+/**
+ * Internal dependencies
+ */
+import { unlock } from '../../lock-unlock';
+
+const { createPrivateSlotFill } = unlock( componentsPrivateApis );
```

Comment:
Note: A similar helper was intentionally removed in the past - https://github.com/WordPress/gutenberg/pull/67238.